src/NHibernate.DomainModel/Northwind/Mappings/User.hbm.xml

```diff
@@ -7,11 +7,19 @@
       <generator class="assigned" />
     </id>
 
-    <property name="Name" type="AnsiString" />
+    <property name="Name" type="AnsiString" not-null="true" />
     <property name="InvalidLoginAttempts" type="Int32" />
     <property name="RegisteredAt" type="DateTime" />
     <property name="LastLoginDate" type="DateTime" />
 
+    <many-to-one name="CreatedBy" class="User" not-null="true" lazy="false">
```

Comment:
I had to add `lazy="false"`, otherwise some tests in `QueryReuseTests` (`CanReuseAfterFirst`, `CanReuseAfterFirstOrDefault`, `CanReuseAfterSingle`,  `CanReuseAfterSingleOfDefault`) would fail.